CircleCI Partner Review

Hello Autify team 👋

I am from the @circleci Community and Partner Engineering team, here to help with a review for the CircleCI orb partnership. I have an action item below for you:

- Double quote variables

The variable at the line [5](https://github.com/autifyhq/autify-circleci-orb-cli/blob/main/src/scripts/install.bash#L5) should be double-quoted to comply with [SC2086](https://www.shellcheck.net/wiki/SC2086).

---
Feel free to ask me any questions, and please include me as a reviewer in the PR with the changes. Thank you!

--- a/.circleci/config.yml
+++ b/.circleci/config.yml
@@ -18,7 +18,6 @@ workflows:
       - orb-tools/review:
           filters: *filters
       - shellcheck/check:
-          exclude: SC2148,SC2038,SC2086,SC2002,SC2016
           filters: *filters
       - orb-tools/publish:
           orb-name: autify/autify-cli

--- a/src/commands/install.yml
+++ b/src/commands/install.yml
@@ -12,4 +12,4 @@ steps:
       environment:
         PARAM_VERSION: <<parameters.version>>
       name: Install Autify CLI
-      command: <<include(scripts/install.bash)>>
+      command: <<include(scripts/install.sh)>>

--- a/src/scripts/install.sh
+++ b/src/scripts/install.sh
@@ -2,5 +2,5 @@
 curl https://autify-cli-assets.s3.amazonaws.com/autify-cli/channels/stable/install-standalone.sh | bash
 
 if [ "$PARAM_VERSION" != "stable" ]; then
-  autify update -v $PARAM_VERSION
+  autify update -v "$PARAM_VERSION"
 fi

